books/RayTracingInOneWeekend.html

```diff
@@ -2332,7 +2351,7 @@
 
 The value of $\sin\theta'$ cannot be greater than 1. So, if,
 
-  $$ \frac{1.5}{1.0} \cdot \sin\theta > 1.0 $$,
+  $$ \frac{1.5}{1.0} \cdot \sin\theta > 1.0 $$
```

Comment:
This is standard notation, we should keep the comma.

Reply:
Agree that the comma is grammatically correct, but it looks _WEIRD_

Reply:
Oh, I'm thinking of somewhere else, will revert

Reply:
Oh man you're right — it does _not_ belong on a line by itself. Put the comma inside the `$$` delimiters and tell me what you think. Looks good to me.